go/teams/service_helper.go

```diff
@@ -82,6 +60,8 @@ func Details(ctx context.Context, g *libkb.GlobalContext, name string) (res keyb
 	}
 	res.AnnotatedActiveInvites = annotatedInvites
 
+	membersHideInactiveDuplicates(ctx, g, &res.Members)
```

Comment:
Do you think these two are mutually exclusive?
- Re-adding post-reset owners to a team should work
- Having an invariant that the same uid is not present in a team twice including conflicts between sigchain-invites and cryptomembers

Reply:
> Anyway it disallows having multiple versions of the same user in the same team on database assertion level.
Does this consider sigchain-invites to conflict with cryptomembers? As of now this PR shouldn't cause conflicting cryptomembers. But there is a path where there could be a cryptomember eee%8 and a sigchain-invite for eee%0.

Reply:
I think it would be a conflict, it's been a while but what I'm seeing now is that we are relying on key on `team_members_union` table, which holds both crypto and pukless members.

Reply:
I've observed the server letting it through. Though idk if it causes other non-obvious issues and it also lets through other stuff it shouldn't. What should be done?

Reply:
oh cause it's disabled by default. look for `strict_union_inserts`